releases.yml

```diff
@@ -54,6 +54,10 @@
       url: /docs/patterns/tiered-list
       status: New
       notes: We've introduced a new tiered list pattern.
+    - component: Equal Height Row
+      url: /docs/patterns/equal-height-row#wrap-variant
+      status: New
+      notes: We've added a new wrap variant to the equal height row component.
```

Comment:
This has been added to `4.15.0` release notes section, but it should be added to `4.16.0`.

<img width="989" alt="Screenshot 2024-08-22 at 5 33 29 PM" src="https://github.com/user-attachments/assets/a5a69205-f44d-4807-8e33-427f493f33b6">